ts/Dashboards/EditMode/EditRenderer.ts

```diff
@@ -320,6 +388,119 @@ function renderText(
     return textElem;
 }
 
+function renderNested(
+    parentElement: HTMLDOMElement,
+    options: any
+): HTMLDOMElement|undefined {
+
+    if (!parentElement) {
+        return;
+    }
+    const keys = Object.keys(options.nestedOptions);
+    for (let i = 0, iEnd = keys.length; i < iEnd; ++i) {
+        const name = keys[i];
+        const nestedOptions = options.nestedOptions[name];
+
+
+        const nested = createElement(
+            'div',
+            {
+                className: 'highcharts-dashboards-nested'
+            },
+            {},
+            parentElement
+        );
+
+        const header = createElement(
+            'div',
+            {
+                className: 'highcharts-dashboards-nested-header'
+            },
+            {},
+            nested
+        );
+        const headerBtn = createElement(
+            'button',
+            { className: 'highcharts-dashboards-nested-header-btn' },
+            {
+                border: 'none',
+                font: 'inherit',
+                color: 'inherit',
+                background: 'none',
+                margin: 0,
+                width: '100%',
+                display: 'flex'
+            },
+            header
+        );
+
+        const headerIcon = createElement(
+            'img',
+            {
+                className: 'highcharts-dashboards-nested-header-icon',
+                src: Globals.iconsURLPrefix + 'dropdown-pointer.svg'
+            },
+            {},
+            headerBtn
+        );
+
+
+        createElement(
+            'span',
+            { textContent: name },
+            {},
+            headerBtn
+        );
+
+        const switchElement = createElement(
```

Comment:
Do we need a variable for each that button? You are not using it later anywhere

Reply:
It will later be used by toggle, so I will leave it as it is for now.